src/cpp/src/visual_language/phi3_vision/classes.cpp

```diff
@@ -66,11 +66,11 @@ ov::Tensor HD_transform(const ov::Tensor& uint8, size_t num_crops) {
     if (trans) {
         src = clip_image_u8{int(height), int(width), {uint8_data, uint8_data + uint8.get_size()}};
         bilinear_resize(src, dst, new_h, new_w);
-        return padding_336(ov::Tensor{ov::element::u8, {1, new_w, new_h, 3}, dst.buf.data()});
+        return padding_336(ov::Tensor{ov::element::u8, {1, new_w, new_h, 3}, const_cast<uint8_t*>(dst.buf.data())});
     }
     src = clip_image_u8{int(width), int(height), {uint8_data, uint8_data + uint8.get_size()}};
     bilinear_resize(src, dst, new_w, new_h);
-    return padding_336(ov::Tensor{ov::element::u8, {1, new_h, new_w, 3}, dst.buf.data()});
+    return padding_336(ov::Tensor{ov::element::u8, {1, new_h, new_w, 3}, const_cast<uint8_t*>(dst.buf.data())});
```

Comment:
Maybe the `dst.buf.data()` should be not a const.
The const_cast looks like WA and maybe root of problem should be fixed if not possible because some external lib API issue is good to comment why source is const and we remove it for modification.